Integrations/BeyondTrust_Password_Safe/BeyondTrust_Password_Safe.py

```diff
@@ -0,0 +1,316 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+''' IMPORTS '''
+
+import json
+import requests
+from requests import Request, Session
```

Comment:
why are you importing `Request` for?
don't import `requests` and `from requests`